core/storage/suggestion/gae_models.py

```diff
@@ -2396,9 +2401,15 @@ def fetch_page(
             TranslationSubmitterTotalContributionStatsModel] = []
         today = datetime.date.today()
 
-        if max_days_since_last_activity is not None:
+        if max_days_since_start_date_for_last_activity is not None:
```

Comment:
This is not the right place for this logic. The storage layer should not calculate dates from today, even if it did so before.

You want to get some data for some range - then pass the `range`, a single object of structure with two fields: start_date, end_date. If start_date is empty, fill it as 1970-01-01 somewhere above, so that it's simple to use this structure. If end_date is empty, set it to 3000-01-01.

Reply:
Hello @nikitaevg , made the suggested changes, PTAL